src/applications/vaos/project-cheetah/components/VAFacilityPage/VAFacilityInfoMessage.jsx

```diff
@@ -1,28 +1,30 @@
 import React from 'react';
 import AlertBox from '@department-of-veterans-affairs/component-library/AlertBox';
 import State from '../../../components/State';
-import NewTabAnchor from '../../../components/NewTabAnchor';
 
-export default function VAFacilityInfoMessage({ facility }) {
+export default function VAFacilityInfoMessage({ facility, sortMethod }) {
   return (
-    <AlertBox
-      status="info"
-      headline="We found one VA location for you"
-      level="2"
-    >
+    <div className="vads-u-margin-bottom--4">
       <p>
-        <strong>{facility.name}</strong>
-        <br />
-        {facility.address?.city}, <State state={facility.address?.state} />
+        We found one VA location where you're registered that offer COVID-19
```

Comment:
I'm guessing this is copied over, but I think it should be "offers" instead of "offer". It'd be good to fix that in both places.

Reply:
Good catch, updated. Correct me if I'm wrong but I think the multiple facilities list wording should stay `offer` since it's referring to multiple locations.

Reply:
That's correct, but I believe there's an equivalent component to this in the `new-appointment` folder that is only showing when we have one facility to present.

Reply:
Ah understood, I didn't update those components because their title infos are worded differently. 

In `new-appointment/components/VAFacilityPage/SingleFacilityAvailable.jsx`: 

> _"We found one VA facility for your example-care appointment."_

and in `new-appointment/components/VAFacilityPage/VAFacilityInfoMessage.jsx`: 

> _"We found one facility that accepts online scheduling for this care"_